framework/src/loops/UpdateDisplacedMeshThread.C

```diff
@@ -96,6 +96,8 @@ UpdateDisplacedMeshThread::init()
         soln->size(), soln->local_size(), send_list.send_list(), true, libMesh::GHOSTED);
     soln->localize(*ghost_soln, send_list.send_list());
   }
+
+  _has_displacement = false;
```

Comment:
Set this in the constructor instead

Reply:
Will add that. But I think we should keep this as well, in case someone call this multiple times.